### PR TITLE
Init commit

### PR DESCRIPTION
**Pull request by mnecas**
First commit contains:
- automation for Jenkins
- sdk
- readme
- examples
- spec files
- tests

### CODE REVIEW

**Comment by mnecas** [comment]
ci test please

File: pom.xml [hunk 1]
```diff
@@ -31,136 +31,4 @@ limitations under the License.
```
Comment (mwperina) [inline pom.xml:31]: pom.xml should be removed from the project, it should be contained only in the generator project

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mwperina) [inline pom.xml:1]: This file should be removed completely

**Review by mwperina — CHANGES_REQUESTED** [review]

**Review by mwperina — APPROVED** [review]
+1

**Pull request merged**